web-common/src/features/entity-management/WatchResourcesClient.ts

```diff
@@ -181,6 +182,15 @@ export class WatchResourcesClient {
             // The following invalidations are only needed if the Source/Model has an active table
             if (!connectorName || !tableName) return;
 
+            // If it's a new source, show the "Source imported successfully" modal
+            const isNewSource =
+              res.name.kind === ResourceKind.Source &&
+              res.resource.meta.specVersion === "1";
```

Comment:
Is this check enough? I remember we would sometimes get `2` as well. That is why the earlier implementation checked whether we are seeing the resource for the 1st time.

Reply:
I believe so. We have this condition above it...
```
// The following invalidations are only needed if the Source/Model has an active table
if (!connectorName || !tableName) return;
```
... which ensures the source reconciler successfully yielded a table. It's at this point that `stateVersion === "2"`, but I don't think a check for that is important.